test/test_native_transfer.py

```diff
@@ -0,0 +1,59 @@
+from gql import gql
+from functools import reduce
+import time, unittest, base
+
+
+class TestTransfer(base.Base):
+    amount = 5000000
+    denom = "atestfet"
+    msg_type = '/cosmos.bank.v1beta1.MsgSend'
+    db_query = 'SELECT * from native_transfers'
+
+    def test_native_transfer(self):
+        self.db_cursor.execute('TRUNCATE table native_transfers')
+        self.db.commit()
+        self.assertFalse(self.db_cursor.execute(self.db_query).fetchall(), "\nError: table not empty after truncation")
+
+        tx = self.ledger_client.send_tokens(self.delegator_wallet.address(), self.amount, self.denom, self.validator_wallet)
+        tx.wait_to_complete()
+        self.assertTrue(tx.response.is_successful(), "\nTXError: transfer tx unsuccessful")
+
+        time.sleep(5)
```

Comment:
It may be worth adding a comment (and maybe a TODO too :sweat_smile:) like:
```suggestion
        # TODO: something more robust
        # Wait for the indexer
        time.sleep(5)
```